app/src/main/AndroidManifest.xml

```diff
@@ -8,8 +8,10 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.HackTziBlog">
-        <activity android:name=".MainActivity">
+       >
+        <activity android:name=".MainActivity"
+            android:theme="@style/SplashTheme"
```

Comment:
Do not forget to add screenOrientation attribute in order to avoid rotation. For now we only support portrait :)